docs/sources/tempo/metrics-from-traces/metrics-queries/configure-traceql-metrics.md

```diff
@@ -135,3 +135,31 @@ query_frontend:
     concurrent_jobs: 8
     target_bytes_per_job: 1.25e+09 # ~1.25GB
 ```
+
+## Sampling and performance optimization
+
+TraceQL metrics queries support sampling hints to improve performance on large datasets.
+
+### Sampling configuration considerations
+
+When using sampling in your TraceQL metrics queries, consider:
+
+- **Timeout settings:** Sampled queries run faster but may still benefit from adequate timeouts
+- **Concurrent jobs:** Sampling reduces per-job processing time, allowing higher concurrency
+- **Job sizing:** With sampling, smaller job sizes may be more efficient
+
+Example configuration optimized for sampling:
+
+```yaml
+query_frontend:
+  metrics:
+    concurrent_jobs: 1500 # Higher concurrency with sampling
+    target_bytes_per_job: 1.5e+08 # Smaller jobs with sampling
+```
```

Comment:
Is this example correct?

Reply:
This is kind of interesting, the notes here about tunables that may be affected by sampling kind of makes sense, but totally untested and speculative. Need to be removed.